booking/src/main/java/org/dcsa/conformance/standards/booking/checks/ShipperBookingContentConformanceCheck.java

```diff
@@ -39,61 +41,63 @@ private boolean isReeferContainerSizeTypeCode(String isoEquipmentCode) {
   }
 
   protected Set<String> reeferChecks(JsonNode payload) {
-    var requestedEquipments = payload.path("requestedEquipments");
     var issues = new LinkedHashSet<String>();
-    int index = -1;
-    for (var requestedEquipment : requestedEquipments) {
-      var code = requestedEquipment.path("ISOEquipmentCode").asText(null);
-      index++;
-      var prefix = "requestedEquipments[" + index + "].";
-      if (code == null || code.length() < 4) {
-        // Schema validation should catch this case.
-        continue;
-      }
-      if (isReeferContainerSizeTypeCode(code)) {
-        var node = fieldRequired(
-          requestedEquipment,
-          IS_NOR_FIELD,
-          issues,
-          prefix,
-          "the ISOEquipmentCode implies that the container is a reefer container"
-        );
-        if (node != null && node.isBoolean()) {
-          var isNOR = node.booleanValue();
-          if (!isNOR) {
-            fieldRequired(
-              requestedEquipment,
-              ACTIVE_REEFER_SETTINGS_FIELD,
-              issues,
-              prefix,
-              "the ISOEquipmentCode implies that the container is a reefer container" +
-                " and isNonOperatingReefer was false"
-            );
-          } else {
-            fieldOmitted(
-              requestedEquipment,
-              ACTIVE_REEFER_SETTINGS_FIELD,
-              issues,
-              prefix,
-              "the isNonOperatingReefer was true"
-            );
+    if (!scenarioType.equals(ScenarioType.REGULAR_NON_OPERATING_REEFER)) {
```

Comment:
I do not understand why you are making this validation conditional on the scenario. The validation here was not intended to be scenario specific.

Reply:
I have created a story to add the scenario based checks DT-897